dipy/segment/tissue.py

```diff
@@ -125,3 +126,115 @@ def classify(self, image, nclasses, beta, *, tolerance=1e-05, max_iter=100):
         PVE = PVE[..., 1:]
 
         return initial_segmentation, final_segmentation, PVE
+
+
+def compute_directional_average(data, bvals, *,
+                                b0_thresh=50, low_signal_threshold=50):
+    """
+    Compute the mean signal for each unique b-value shell and fit a linear model.
+
+    Parameters
+    ----------
+    data : ndarray
+        The diffusion MRI data.
+    bvals : ndarray
+        The b-values corresponding to the diffusion data.
+    b0_thresh : float, optional
+        The threshold below for a b=0 image.
+        Default is 50.
```

Comment:
Please, avoid saying here what the default value is: it is clear from the method signature. If the value in the signature is changed for whatever reasons, the risk of not updating the parameter docstring value is high, and so the documentation would go out of sync, be misleading, and require more maintenance. Remove all values except for those cases where the default value requires some coding in the method making the parameter take a value that is not the same as the one in the signature.

Reply:
Yeah. Makes sense. Thank you for the comment. Will remove it asap